product_docs/docs/edbcloud/beta/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Modifying Database Configuration Parameters
+---
+
+The database parameters listed on the DB Configuration tab are also referred to as Grand Unified Configuration (GUC) variables. See [What Is a GUC Variable?](https://www.enterprisedb.com/blog/what-guc-variable) for more information.
+
+The list of parameters is populated based on the type of database you selected on the **Operational Settings** tab when you created your cluster. For more information about the parameters for your database type:
+
+- For Postgres parameters, see 
+   - [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
+   - [postgresqlco.nf](https://postgresqlco.nf/) for more documentation and a tool to manage your configuration file.
```

Comment:
Two things on this:

1. Most of the parameters here are also valid for EPAS - we shouldn't sub-bullet this under PostgreSQL.
2. "and a tool to manage your configuration file" - customers can't use this tool with EDB Cloud.

I won't object to removing postgresqlco.nf if we can't figure out how to reference it nicely.